src/01-1-basics.js

```diff
@@ -17,10 +29,17 @@ function gradeCommentary() {
 
 function logNumbers() {
   // 1. Log numbers from 1 to 10 using a for loop.
+  for (let i = 1; i <= 10; i++) {
+    console.log(i);
+  }
 }
 
 function countByTwos() {
   // 2. Using a while loop, log even numbers from 2 to 20.
+  let i = 2;
+    while (i <= 2) {
+      console.log(i)
```

Comment:
you need to increment i here something like `i+=2`

right now you have an 'infinite loop' scenario which causes the remote testing to fail!